client/stylesheets/_toCat.scss

```diff
@@ -5604,12 +5608,14 @@ On click the images will be displayed at normal size to complete the effect
     padding: 10px 15px;
     color: #fff !important;
     border-radius: 4px;
-    background-color: #404040 !important;
+    background-color: #f8504b !important;
```

Comment:
Color literals like `#f8504b` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces